selfdrive/car/gm/interface.py

```diff
@@ -135,6 +146,46 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=None, disa
       ret.lateralTuning.pid.kf = 0.000045
       tire_stiffness_factor = 1.0
 
+    elif candidate == CAR.SILVERADO:
+      ret.minEnableSpeed = -1.
+      ret.mass = 2400. + STD_CARGO_KG
+      ret.wheelbase = 3.745
+      ret.steerRatio = 16.3
+      ret.centerToFront = ret.wheelbase * .49
+      ret.networkLocation = NetworkLocation.fwdCamera # Uses Cam Harness
+      ret.radarOffCan = True # No Radar
+      ret.openpilotLongitudinalControl = False # Stock ACC
+      ret.pcmCruise = True # CC is on
+      # Tune
```

Comment:
Note: This tune needs review. Skip is now using a torque tune with some "fancy" customization for daily driving. Working to simplify still